ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -379,7 +379,7 @@ bb_setOption (void **pdata, int opNum, const char *param)
     bounding_box_modes previous = bdata->mode;
     bdata->mode = find_key_strv (bb_modes, param);
 
-    if (NULL == param || *param == '\0' || bdata->mode < 0) {
+    if (NULL == param || *param == '\0') {
```

Comment:
Uh.. why?

Reply:
@myungjoo This enum, bounding_box_modes, is started from 0. 
```bash
/nnstreamer-1.0.0/ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

/private/tmp/nnstreamer-20191003-458-yyl8bx/nnstreamer-1.0.0/ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:382:56: error: comparison of unsigned enum expression < 0 is always false [-Werror,-Wtautological-compare]

    if (NULL == param || *param == '\0' || bdata->mode < 0) {

                                           ~~~~~~~~~~~ ^ ~

1 error generated.

[55/100] Linking static target ext/nnstreamer/tensor_decoder/libnnstreamer_decoder_image_labeling.a.

[56/100] Compiling C object 'ext/nnstreamer/tensor_decoder/7bf3534@@nnstreamer_decoder_bounding_boxes@sta/tensordec-boundingbox.c.o'.

FAILED: ext/nnstreamer/tensor_decoder/7bf3534@@nnstreamer_decoder_bounding_boxes@sta/tensordec-boundingbox.c.o 

clang -Iext/nnstreamer/tensor_decoder/7bf3534@@nnstreamer_decoder_bounding_boxes@sta -Iext/nnstreamer/tensor_decoder -I../ext/nnstreamer/tensor_decoder -Igst/nnstreamer -I../gst/nnstreamer -I/usr/local/Cellar/pcre/8.43/include -I/usr/local/Cellar/glib/2.62.0_1/include/glib-2.0 -I/usr/local/Cellar/glib/2.62.0_1/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/libffi/3.2.1/lib/libffi-3.2.1/include -I/usr/local/Cellar/glib/2.62.0_1/include -I/usr/local/Cellar/gstreamer/1.16.1/include/gstreamer-1.0 -I/usr/local/Cellar/orc/0.4.29/include/orc-0.4 -I/usr/local/Cellar/gst-plugins-base/1.16.1/include/gstreamer-1.0 -I/System/Library/Frameworks/JavaVM.framework/Versions/Current/Headers/ -Xclang -fcolor-diagnostics -pipe -Wall -Winvalid-pch -Werror -std=gnu89 -g '-DVERSION="1.0.0"' -D__MACOS__=1 -Wredundant-decls -Wwrite-strings -Wformat -Wformat-nonliteral -Wformat-security -Winit-self -Waddress -Wno-multichar -Wvla -Wpointer-arith -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Waggregate-return -Wold-style-definition -Wdeclaration-after-statement '-DNNSTREAMER_CONF_FILE="/usr/local/Cellar/nnstreamer/1.0.0/etc/nnstreamer.ini"' -DHAVE_ORC=1 -DENABLE_TENSORFLOW=1 -MD -MQ 'ext/nnstreamer/tensor_decoder/7bf3534@@nnstreamer_decoder_bounding_boxes@sta/tensordec-boundingbox.c.o' -MF 'ext/nnstreamer/tensor_decoder/7bf3534@@nnstreamer_decoder_bounding_boxes@sta/tensordec-boundingbox.c.o.d' -o 'ext/nnstreamer/tensor_decoder/7bf3534@@nnstreamer_decoder_bounding_boxes@sta/tensordec-boundingbox.c.o' -c /private/tmp/nnstreamer-20191003-458-yyl8bx/nnstreamer-1.0.0/ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

/private/tmp/nnstreamer-20191003-458-yyl8bx/nnstreamer-1.0.0/ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c:382:56: error: comparison of unsigned enum expression < 0 is always false [-Werror,-Wtautological-compare]

    if (NULL == param || *param == '\0' || bdata->mode < 0) {

                                           ~~~~~~~~~~~ ^ ~

1 error generated.

ninja: build stopped: subcommand failed.
```

clang (not latest) complains about it.

Reply:
Oh.. ok..